spec/indico_batch_spec.rb

```diff
@@ -108,14 +108,36 @@
     expect Set.new(response[1].keys).subset?(Set.new(expected_keys))
   end
 
-  it 'should tag text with correct keywords' do
-    expected_keys = Set.new(%w(people kill guns))
+  # add test when keyword api is added to public cloud
+  #
+  # it 'should tag text with correct keywords' do
+  #   expected_keys = Set.new(%w(people kill guns))
 
-    data = ['Guns don\'t kill people.', 'People kill people.']
-    response = Indico.batch_keywords(data, @config)
+  #   data = ['Guns don\'t kill people.', 'People kill people.']
+  #   response = Indico.batch_keywords(data, @config)
 
-    expect Set.new(response[0].keys).subset?(Set.new(expected_keys))
-    expect Set.new(response[1].keys).subset?(Set.new(expected_keys))
+  #   expect Set.new(response[0].keys).subset?(Set.new(expected_keys))
+  #   expect Set.new(response[1].keys).subset?(Set.new(expected_keys))
+  # end
+
+  it 'should return all named entities with category breakdowns' do
+    expected_keys = Set.new(%w(unknown organization location person))
+    response = Indico.batch_named_entities(['I want to move to New York City!'])
+    expect(response.length).to eql(1)
+    response = response[0]
+
+    expect(response.keys.length > 0).to eql(true)
+    expect(response.values[0]['confidence'] >= 0.75).to eql(true)
+    expect(Set.new(response.values[0]['categories'].keys)).to eql(expected_keys)
+
+    chance_sum = response.values[0]['categories'].values.inject{|sum,x| sum + x }
+    expect(chance_sum > 0.9999).to eql(true)
```

Comment:
we could do stuff like `expect(target).to be < value` also. Don't know if it matters too much